classes/class-wc-connect-nux.php

```diff
@@ -90,14 +99,46 @@ public function register_add_service_to_zone_pointer( $pointers ) {
 				'options' => array(
 					'content' => sprintf( '<h3>%s</h3><p>%s</p>',
 						__( 'Add a WooCommerce shipping service to a Zone' ,'woocommerce-services' ),
-						__( 'To ship products to customers using USPS or Canada Post, you will need to add them as a shipping method to an applicable zone. If you don\'t have any zones, add one first.', 'woocommerce-services' )
+						__( "To ship products to customers using USPS or Canada Post, you will need to add them as a shipping method to an applicable zone. If you don't have any zones, add one first.", 'woocommerce-services' )
 					),
 					'position' => array( 'edge' => 'right', 'align' => 'left' ),
-				)
+				),
 			);
 			return $pointers;
 		}
 
+		public function is_new_labels_user() {
+			$is_new_user = get_transient( self::TRANSIENT_IS_NEW_LABEL_USER );
+			if ( ! is_string( $is_new_user ) ) {
+				global $wpdb;
+				$query = "SELECT meta_key FROM {$wpdb->postmeta} WHERE meta_key = 'wc_connect_labels' LIMIT 1";
+				$results = $wpdb->get_results( $query );
+				$is_new_user  = 0 === count( $results ) ? 'yes' : 'no';
+				set_transient( self::TRANSIENT_IS_NEW_LABEL_USER, $is_new_user );
```

Comment:
What do you think about short-circuiting the db query if we already got a value of > 1 at least once? If we get a value of > 1 (e.g. `$is_new_user` is 'no'/false etc) then we can dismiss the pointer using code, like this:

`add_user_meta( get_current_user_id(), 'dismissed_wp_pointers', 'wc_services_labels_metabox' );`

And check for the dismissal in `register_order_page_labels_pointer` using:

`get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true )`

That way, even when the transient expires we don't have to check the db again.